src/NServiceBus.Metrics.AcceptanceTests/Tests/When_reporting_to_ServiceControl.cs

```diff
@@ -0,0 +1,70 @@
+﻿namespace NServiceBus.Metrics.AcceptanceTests
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using AcceptanceTesting.Customization;
+    using global::Metrics.Json;
+    using NServiceBus.AcceptanceTests;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NUnit.Framework;
+
+    public class When_reporting_to_ServiceControl : NServiceBusAcceptanceTest
+    {
+        static string MonitoringSpyAddress => Conventions.EndpointNamingConvention(typeof(MonitoringSpy));
+
+        [Test]
+        public async Task Should_send_reports_to_configured_queue()
```

Comment:
I think this should be a separate test that checks that. Here I only wanted to check if the communication works and the data can be deserialized. Contents (such as which metrics are included) should be verified elsewhere.